dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/metadata/objectbundle/ObjectBundleServiceAttributesTest.java

```diff
@@ -280,13 +280,16 @@ private void defaultSetupWithAttributes()
         DataElement de2 = createDataElement( 'B' );
         DataElement de3 = createDataElement( 'C' );
 
-        attributeService.addAttributeValue( de1, attributeValue1 );
-        attributeService.addAttributeValue( de2, attributeValue2 );
-        attributeService.addAttributeValue( de3, attributeValue3 );
-
-        manager.save( de1 );
-        manager.save( de2 );
-        manager.save( de3 );
+        transactionTemplate.execute( status -> {
+            manager.save( de1 );
+            manager.save( de2 );
+            manager.save( de3 );
+            attributeService.addAttributeValue( de1, attributeValue1 );
+            attributeService.addAttributeValue( de2, attributeValue2 );
+            attributeService.addAttributeValue( de3, attributeValue3 );
+            manager.clear();
```

Comment:
FYI: for some reason this needed forcing so that updates are running now and not further deferred.